package.json

```diff
@@ -2,26 +2,47 @@
   "name": "idbify",
   "version": "1.0.0",
   "description": "A simple interface for IDB with promises, you can define your schema up front and then it provides put, delete, and get functions.",
-  "main": "dist/idbify.es5.js",
-  "module": "dist/idbify.min.js",
+  "main": "dist/idbify.umd.js",
+  "module": "dist/idbify.module.js",
   "repository": "https://github.com/coltpini/idbify.git",
   "author": "Colt Pini <coltpini@github.com>",
   "license": "Apache 2.0",
+  "babel": {
```

Comment:
Is there a benefit of keeping babel configuration here as opposed to on its own in `babelrc`?

Reply:
Just one less file to maintain.